src/main/java/com/hida/ArkMinter.java

```diff
@@ -0,0 +1,251 @@
+package com.hida;
+
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.LinkedList;
+import javax.json.Json;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+/**
+ * A class that creates ids using the ark formatting convention. Currently
+ * deciding whether or not to merge this with Minter
+ * @author Brittany Cruz
+ */
+public class ArkMinter extends Minter {
+
+    // fields
+    private final String NAAN = "/:ark/70111/";
+    private final int AMOUNT;
+    private final int LENGTH;
+    private final String PREFIX;
+    private final String FORMAT;
+    private final LinkedList<String> ID_ARRAY = new LinkedList();
+    private final HashMap<String, String> tokenMaps = new HashMap();
+
+    /**
+     * Instantiates an Ark minter that will generate the requested amount of
+     * ids
+     * @param amount - amount of ids to mint
+     */
+    public ArkMinter(int amount) {
+        retrieveSettings();
+        this.AMOUNT = amount;
+        this.PREFIX = getIdPrefix();
+        this.FORMAT = getCharMap();
+        this.LENGTH = (10 - this.PREFIX.length());
+
+        /*
+        deciding whether or not to store these values in a database or just 
+        constantly generate them. If the latter, should I store them 
+        somewhere else, such as in a separate class or in the minter        
+        */
+        this.tokenMaps.put("DIGIT", "0123456789");

```

Comment:
ATM, having them in a map doesn't seem to give you anything..separate `String`s would do just fine.